docs/source/contribute/development/developer_guide/release_instructions.rst

```diff
@@ -0,0 +1,182 @@
+.. _release:
+
+Release Management
+=====================================
+
+This section provides detailed instructions used to release a new version of ``skbase``.
+
+This task is carried out by core developers with write access to the ``skbase``
+repository and designated as release managers by the
+:ref:`Community Council <cc-members>`.
+
+Summary of release process
+--------------------------
+
+The release process includes, in sequence:
+
+* :ref:`prepare for an upcoming release <cycle_process>`
+* :ref:`get the release ready <release_version_prep>`
+* :ref:`release on PyPi <pypi_release>`
+* :ref:`release on conda <conda_release>`
+* troubeleshooting and accident remediation, if applicable (see troubeleshooting
+  tips in each of the prior sections)
+
+Details follow below.
+
+.. _cycle_process:
+
+Release preparation cycle
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``skbase`` aims for a release every month, typically coinciding with the start of
+the month. To ensure releases go smoothly, the following steps are taken leading
+up to each release:
+
+1. 1 week before release date, update the release project board and alert
+   project contributors of upcoming release on slack.
+2. For major releases or substantial features, optionally extend the release cycle,
+   if needed, so that the changes can be completed and incorporated in the release.
+   If a release will be delayed, notify project contributors on slack.
+3. All changes to the main branch of the repository are frozen 1 day prior to the
+   release. At this point only the release managers (for this release) should
+   merge any Pull Requests. Remind core developers of the timing of the feature
```

Comment:
odd capitalization of Pull Request

Reply:
Fixed, I guess I was using it as a proper noun, but it isn't really.